packages/api/src/auth.js

```diff
@@ -146,14 +147,34 @@ export function withPinningAuthorized (handler) {
  */
 async function tryMagicToken (token, env, bypass = magicLinkBypass) {
```

Comment:
maybe `bypass` should still be `dangerouslyBypass`

Reply:
`magicLinkBypass` should be defined in this file so it's front and center

Reply:
after this commit, the (now two) bypasses are all prefixed with `dangerous` https://github.com/web3-storage/web3.storage/pull/1754/commits/ded465b5f949dfad96b89890681d46bef79f613a#diff-d8c9d6f9ae3c9c310454275870e4b2b397b36411752f664a0540d301027bbc1dR157